### DISCUSSION TITLE
Is it possible to prevent test fuzzing from sending 0 input?

### DISCUSSION THREAD

**Discussion by 0xFloop** [foundry-rs/foundry#1767] [Help] [answered]
I want to fuzz a test, but the test has a 0 value check in the beginning so obviously the fuzzing test always fails when it passes in 0. Is it possible to write test to fuzz but not try 0 as an input?

**Comment by odyslam** [comment, marked as answer]
I think the `vm.assume(x>0)` is the cheatcodee you are searching

**Reply by 0xFloop** [reply]
this is perfect thank you very much
